cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -470,3 +470,13 @@ func (o *objectGraph) setCRSTenant(node, tenant *node) {
 		}
 	}
 }
+
+// warnVirtualNode logs warning if some node are still virtual
+func (o *objectGraph) checkVirtualNode() {
+	log := logf.Log
+	for _, node := range o.uidToNode {
+		if node.virtual {
+			log.Info("node wont be moved because not included in GVK considered for move", "kind", node.identity.Kind, "name", node.identity.Name)
```

Comment:
We can make this a level 1 log since it provides more info about the internal workflow of what's happening and it is not critical that the user needs to know about.

I'm referencing https://github.com/kubernetes-sigs/cluster-api/blob/master/cmd/clusterctl/log/doc.go as guidance.

```suggestion
			log.V(1).Info("Object won't be moved because it's not included in GVK considered for move", "kind", node.identity.Kind, "name", node.identity.Name)
```

Reply:
done, thanks